lib/tpm2_openssl.c

```diff
@@ -1271,3 +1274,118 @@ tpm2_openssl_load_rc tpm2_openssl_load_private(const char *path,
 
     return rc;
 }
+
+bool tpm2_openssl_import_keys(
+    TPM2B_PUBLIC * parent_pub,
+    TPM2B_SENSITIVE * private,
+    TPM2B_PUBLIC * public,
+    const char * input_key_file,
+    TPMI_ALG_PUBLIC key_type,
+    const char * auth_key_file,
+    const char * policy_file,
+    const char * key_auth_str,
+    char * attrs_str,
+    const char * name_alg_str
+)
+{
```

Comment:
```suggestion
bool tpm2_openssl_import_keys(TPM2B_PUBLIC * parent_pub,
    TPM2B_SENSITIVE * private, TPM2B_PUBLIC * public,
    const char * input_key_file, TPMI_ALG_PUBLIC key_type,
    const char * auth_key_file, const char * policy_file,
    const char * key_auth_str, char * attrs_str, const char * name_alg_str) {
```

Reply:
When there are a dozen arguments it seems like that formatting becomes more challenging to read and identify the ordering.